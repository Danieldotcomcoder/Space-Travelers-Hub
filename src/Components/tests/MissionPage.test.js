import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../Redux/configureStore';
import Missions from '../MissionPage';

it('Renders Mission Page', () => {
  const missionpage = renderer.create(
    <Provider store={store}>
      <Missions />
      ,
    </Provider>,
  ).toJSON();
  expect(missionpage).toMatchSnapshot();
});
