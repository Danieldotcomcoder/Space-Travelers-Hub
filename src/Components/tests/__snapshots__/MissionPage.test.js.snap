// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Mission Page 1`] = `
Array [
  <table>
    <thead>
      <tr>
        <td
          className="mission-d"
        >
          Mission
        </td>
        <td
          className="mission-d"
        >
          Description
        </td>
        <td
          className="mission-d"
        >
          Status
        </td>
        <td>
          Join
        </td>
      </tr>
    </thead>
    <tbody />
  </table>,
  ",",
]
`;
